### PR TITLE
build without OpenSSL. 

### PR DESCRIPTION
**Pull request by arif-pragmasys**
Only curve25519 and ed25519 supported

### CODE REVIEW

File: openbsd-compat/arc4random.c [hunk 1]
```diff
@@ -113,8 +113,13 @@ _rs_stir(void)
 		fatal("Couldn't obtain random bytes (error 0x%lx)",
 		    (unsigned long)ERR_get_error());
 #else
+#ifdef WINDOWS
```
Comment (manojampalam) [inline openbsd-compat/arc4random.c]: Please put a TODO comment here to take care of replacing rand() with a more secure generator.

Comment (manojampalam) [inline openbsd-compat/arc4random.c:116]: Please put a TODO comment here to take care of replacing rand() with a more secure generator. You may look at other TODOs in the sources

**Review by manojampalam — CHANGES_REQUESTED** [review]

File: openbsd-compat/arc4random.c (hunk 1 above)
Comment (arif-pragmasys) [reply openbsd-compat/arc4random.c:116]: TODO comment added

**Comment by manojampalam** [comment]
Manually squashed and merged

**Pull request closed without merging**